slick.remotemodel-yahoo.ts

```diff
@@ -94,7 +94,8 @@
           url: url,
           callbackParameter: "callback",
           cache: true,
-          success: function (json, textStatus, xOptions) {
+          success: function (json, _textStatus, _xOptions) {
```

Comment:
I am not even certain those two prefixed params are needed

Reply:
If they're unused you should be able to drop them.

Reply:
Agreed. I will do that.